ydb/core/tx/schemeshard/ut_login/ut_login.cpp

```diff
@@ -177,6 +192,171 @@ Y_UNIT_TEST_SUITE(TSchemeShardLoginTest) {
         resultLogin = Login(runtime, "user12", "passwordu12*#");
         UNIT_ASSERT_VALUES_EQUAL(resultLogin.error(), "");
     }
+
+    Y_UNIT_TEST(AccountLockoutAndAutomaticallyUnlock) {
+        TTestBasicRuntime runtime;
+        runtime.AddAppDataInit([](ui32 nodeIdx, NKikimr::TAppData& appData) {
+            Y_UNUSED(nodeIdx);
+            auto accountLockout = appData.AuthConfig.MutableAccountLockout();
+            // AttemptThreshold = 4
+            accountLockout->SetAttemptResetDuration("5s");
+        });
+        TTestEnv env(runtime);
+        ui64 txId = 100;
+        CreateAlterLoginCreateUser(runtime, ++txId, "/MyRoot", "user1", "password1");
+        auto resultLogin = Login(runtime, "user1", "wrongpassword1");
```

Comment:
May be `for(size_t attempt = 0 ; attempt++ ; attempt <=5 )` ?

Reply:
done